submissions/akulahere/task-js-post-oop/index.js

```diff
@@ -0,0 +1,65 @@
+class Inhabitant {
+  constructor(species, name, gender, legs, hands, saying) {
+    this.species = species;
+    this.name = name;
+    this.gender = gender;
+    this.legs = legs;
+    this.hands = hands;
+    this.saying = saying;
+    this.friends = Array.isArray(this.friends) ? this.friends : 'No friends' ;
+  }
+
+  makeFriend(friend) {
+    this.friends = this.friends === 'No friends' ? [] : this.friends;
+    this.friends.push(friend.name);
+  }
+
+  makeMessage() {
+    const properties = [];
+    for (const property of Object.keys(this)) {
+      properties.push(property.toString());
+    }
```

Comment:
[This is still valid](https://github.com/kottans/frontend/blob/master/tasks/js-pre-oop.md#additional-requirements)

Reply:
fixed